model/browser/events.yaml

```diff
@@ -52,3 +52,50 @@ groups:
           brief: A unique ID representing this particular metric instance.
           requirement_level: required
           examples: ["v3-1677874579383-6381583661209"]
+  - id: event.browser.resource_timing
+    stability: development
+    type: event
+    name: browser.resource_timing
+    brief: >
+      This event describes the timing metrics as provided by PerformanceResourceTiming Performance API.
+      These metrics are related to fetching a resource, such as
+      XMLHttpRequest, Fetch, sendBeacon APIs, SVG, image or script.
+    note: >
+      This event captures data from the [ResourceTiming](https://www.w3.org/TR/resource-timing/).
+      It is recommended to be captured by using the PerformanceResourceTiming API.
+      If the page starts unloading before the page loads,
+      then the partial data can be captured by calling the [performance.getEntriesByType](https://developer.mozilla.org/en-US/docs/Web/API/Performance/getEntriesByType) method.
+    attributes:
+      - ref: browser.resource_timing.url.canonical
+        requirement_level: required
+        note: >
+          This is the canonical URL of the resource being measured, not the page URL.
+          It represents the resource in its standard form for identification purposes.
+          The canonical URL includes the scheme (if present), full domain, and path only.
+          Query strings and credentials MUST be excluded to reduce cardinality and avoid capturing PII.
+          For example, `https://www.example.com/path?a=b&c=d` would be captured as `https://www.example.com/path`.
+          In JavaScript, this typically corresponds to `${url.origin}${url.pathname}` or `${location.protocol}//${location.host}${location.pathname}`.
+      - ref: browser.resource_timing.url.query
+        requirement_level: opt_in
+        note: >
+          This attribute is opt-in due to potential high cardinality and PII concerns.
+          Sensitive content provided in the query string SHOULD be scrubbed when instrumentations can identify it.
+          Use this attribute when query parameters are relevant to the resource being fetched (e.g., image dimensions, format specifications).
```

Comment:
I pitched `url.canonical` to the larger Browser SIG group today and we agreed it needs more discussion. For now, please revert the name to `url` and drop `url.query` altogether.